src/System.Net.WebSockets/src/System.Net.WebSockets.csproj

```diff
@@ -18,18 +19,31 @@
     <Compile Include="System\Net\WebSockets\WebSocketMessageType.cs" />
     <Compile Include="System\Net\WebSockets\WebSocketReceiveResult.cs" />
     <Compile Include="System\Net\WebSockets\WebSocketState.cs" />
+    <Compile Include="$(CommonPath)\System\Net\WebSockets\ManagedWebSocket.cs">
```

Comment:
Thanks! I still agree with the plan to move it: I was just curious if it's worth conditionally compiling it for UAP / Windows. 
I wouldn't do that to save 30k for Desktop and ILC should handle the compilation step for the final UAP apps and so we should see no increase.